HotelBookingTests/HotelBaseClassTest.cs

```diff
@@ -32,5 +35,32 @@ public void ShouldReturnFalseIfPaymentFailed()
             Assert.That(_hotel.MakePayment(250, 200), Is.EqualTo(true));
             Assert.That(_hotel.MakePayment(250, 00), Is.EqualTo(false));
         }
+
+        [Test]
+        public void ShouldProcessPartialSteps()
```

Comment:
Should have:

This name doesn't tell me much. However the problem with this name comes form test itself and how much this test actually testing.

Reply:
Applied